webhook.js

```diff
@@ -10,11 +10,13 @@ var querystring = require('querystring');
 
 var User = require('./lib/user');
 var Prompt = require('./lib/prompt');
+var Reminder = require('./lib/reminder');
 var DialogState = require('./lib/dialog-state');
 
 // Connect the job scheduler to the job immediately, since it needs to be
 // defined before attempting to resume pending jobs.
 Prompt.setJobHook(DialogState.onShouldSendPrompt);
+Reminder.start(DialogState.onShouldSendReminder);
```

Comment:
There's a part of me that thinks this should still be called `setJobHook` even though it doesn't use the persistent scheduler. Thoughts?